### PR TITLE
Starlight bday blog post

### PR DESCRIPTION
**Pull request by sarah11918**
Adds Hideoo and Reuben as authors, and drafts a birthday blog post for Starlight!

### CODE REVIEW

File: src/content/blog/starlight-turns-one.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+We focused on bringing you even more built-in components that are essential to a docs project. And thanks to our global community of contributors, we are proud to offer a translated docs infrastructure in twenty-seven officially supported languages, out of the box.
+
+Starlight shows what a team of developers passionate about docs can do, and we are thankful that so many of you joined us, brought your passion, and made Starlight even more than we hoped it would be.
+
+## What’s next
+
+Watch the full talk from Astro Together describing Starlight's first year, and join us on the road to our next milestone: v1.0 later this year! We're happy to have you aboard!
+
+<YouTube id="https://www.youtube.com/watch?v=TtRtkTzHVBU" />
```
Comment (delucis) [inline src/content/blog/starlight-turns-one.mdx:40]: The `id` here will need updating with the correct YouTube URL when we have it.

**Review by delucis — CHANGES_REQUESTED** [review]

**Review by yanthomasdev — COMMENTED** [review]
LGTM! Just a small comma nit as usual!

File: src/content/blog/starlight-turns-one.mdx (hunk 1 above)
Comment (sarah11918) [inline src/content/blog/starlight-turns-one.mdx:40]:
```suggestion
<YouTube id="https://youtu.be/j8K-ESJF814" />
```

**Review by clearlyTHUYDOAN — APPROVED** [review]
LGTM!

**Pull request merged**